config/translations/ca/create.users.yml

```diff
@@ -0,0 +1,27 @@
+description: 'Create dummy users for your Drupal 8 application.'
+help: 'The <info>create:users</info> command helps you create dummy users.'
+welcome: 'Welcome to the Drupal users generator'
+arguments:
+  roles: 'Role(s) to be used in user creation'
+options:
+    limit: 'How many users would you like to create'
+    password: 'Password to be set to users created'
+    time-range: 'How far back in time should the users be dated'
+questions:
+    roles: 'Select role(s) to be used on user creation'
+    limit: 'Enter how many users would you like to generate'
+    password: 'Enter the password to be set to users created'
+    time-range: 'How far back in time should the users be dated?'
+    time-ranges:
+        - 'Now'
+        - '1 hour ago'
+        - '1 day ago'
+        - '1 week ago'
+        - '1 month ago'
+        - '1 year ago'
+messages:
+    user-id: 'User Id'
+    roles: 'Roles'
+    usernam: 'Username'
+    created: 'Created Time'
+    created-users: 'Created %s users successfully'
```

Comment:
yml files must end with a new line, this comment applies for all of the translation files added.